### PR TITLE
OTJ: Rakdos, Satoru, Vraska

### PR DESCRIPTION
**Pull request by dracontes**
Tested card scripts for:
- [Rakdos, the Muscle](https://scryfall.com/card/otj/226/rakdos-the-muscle)
- [Satoru, the Infiltrator](https://www.magicspoiler.com/mtg-spoiler/satoru-the-infiltrator/)
- [Vraska, the Silencer](https://scryfall.com/card/otj/237/vraska-the-silencer)

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Name:Rakdos, the Muscle
+ManaCost:2 B B R
+Types:Legendary Creature Demon Mercenary
+PT:6/5
+K:Flying
+K:Trample
+T:Mode$ Sacrificed | ValidPlayer$ You | ValidCard$ Creature.Other | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you sacrifice another creature, exile cards equal to its mana value from the top of target player's library. Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
+SVar:TrigExile:DB$ Dig | DigNum$ X | ChangeNum$ All | ValidTgts$ Player | TgtPrompt$ Choose a player | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ UntilYourNextEndStep | SpellDescription$ Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt:9]: SpellDescription useless

File: forge-gui/res/cardsfolder/upcoming/vraska_the_silencer.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Vraska, the Silencer
+ManaCost:1 B G
+Types:Legendary Creature Gorgon Assassin
+PT:3/3
+K:Deathtouch
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.OppCtrl+nonToken | TriggerZones$ Battlefield | Execute$ TrigChangeZone | OptionalDecider$ You | TriggerDescription$ Whenever a nontoken creature an opponent controls dies, you may pay {1}. If you do, return that card to the battlefield tapped under your control. It's a Treasure artifact with "{T}, Sacrifice this artifact: Add one mana of any color," and it loses all other card types.
+SVar:TrigChangeZone:AB$ ChangeZone | Cost$ 1 | Defined$ TriggeredCard | RememberChanged$ True | Origin$ Graveyard | Destination$ Battlefield | GainControl$ True | Tapped$ True | AnimateSubAbility$ DBAnimate
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vraska_the_silencer.txt:7]: remembered will be handled internally

File: forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt (hunk 1 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt]: Fair and done. 
Looking a bit more widely there do seem to be other scripts that need similar parameters excised: I got this one from Dead Man's Chest.

File: forge-gui/res/cardsfolder/upcoming/vraska_the_silencer.txt (hunk 2 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/vraska_the_silencer.txt]: Indeed it is: fixed.

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/vraska_the_silencer.txt]: cleanup too

File: forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+K:Trample
+T:Mode$ Sacrificed | ValidPlayer$ You | ValidCard$ Creature.Other | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you sacrifice another creature, exile cards equal to its mana value from the top of target player's library. Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
+SVar:TrigExile:DB$ Dig | DigNum$ X | ChangeNum$ All | ValidTgts$ Player | TgtPrompt$ Choose a player | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ UntilYourNextEndStep
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | MayPlayIgnoreType$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
+A:AB$ Pump | Cost$ Sac<1/Creature.Other/another creature> | Defined$ Self | KW$ Indestructible | ActivationLimit$ 1 | SubAbility$ DBTap | SpellDescription$ CARDNAME gains indestructible until end of turn. Tap it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt:11]: description incomplete

**Review by tool4ever — APPROVED** [review]

**Pull request merged**